### PR TITLE
KFSPTS-25523 remove doc requeuer mod

### PR DESCRIPTION
**Pull request by db483**

### CODE REVIEW

File: src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml [hunk 1]
```diff
@@ -392,15 +392,20 @@
...
           p:exceptionMessageJob-ref="exceptionMessageJob"
+          p:delayedAsyncCallJob-ref="delayedAsyncCallJob"
           p:useQuartzScheduling="${use.quartz.scheduling}"/>
 
     <bean id="messageServiceExecutorJobListener"
           class="org.kuali.kfs.ksb.messaging.quartz.MessageServiceExecutorJobListener"/>
 
     <!--
-        This bean is only meant for KSB-message-related processing, so we exclude it
+        These beans are only meant for KSB-message-related processing, so we exclude it
```
Comment (cah292) [inline src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml:402]: Now that the comment references multiple beans, I would suggest one other quick change: "...exclude it" --> "...exclude them"

File: src/test/resources/edu/cornell/kfs/krad/service/impl/AccountDelegateGlobalTest.xml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 <xmlConversionTestCase>
 
 <oldData>
-<maintainableDocumentContents maintainableImplClass="edu.cornell.kfs.coa.document.CuAccountDelegateGlobalMaintainableImpl"><oldMaintainableObject><edu.cornell.kfs.coa.businessobject.CuAccountDelegateGlobal>
+<maintainableDocumentContents maintainableImplClass="org.kuali.kfs.coa.document.AccountDelegateGlobalMaintainableImpl"><oldMaintainableObject><org.kuali.kfs.coa.businessobject.AccountDelegateGlobal>
```
Comment (cah292) [inline src/test/resources/edu/cornell/kfs/krad/service/impl/AccountDelegateGlobalTest.xml:4]: Should we have at least one test case where the pre-conversion side still references the `CuAccountDelegateGlobal` class, to verify that it's being converted to `AccountDelegateGlobal` correctly?

**Review by cah292 — CHANGES_REQUESTED** [review]
Changes look good overall, though I did find a few items that should be addressed before merging.

File: src/main/java/edu/cornell/kfs/sys/batch/service/CuSchedulerService.java [hunk 3]
```diff
@@ -7,4 +7,5 @@ public interface CuSchedulerService extends SchedulerService {
 
     void scheduleExceptionMessageJob(PersistedMessage message, String description);
 
+    void scheduleDelayedAsyncCallJob(PersistedMessage message, String description);
```
[the remark this answers is not in the document]
Comment (db483) [reply src/main/java/edu/cornell/kfs/sys/batch/service/CuSchedulerService.java:10]: Thank you very much for catching this. I only tested this in my local env and did not realize this was an issue.

File: src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml (hunk 1 above)
Comment (db483) [reply src/main/resources/edu/cornell/kfs/sys/cu-spring-sys.xml:402]: I will update

**Comment by db483** [comment]
@cah292 I made the requested changes and this is ready for review again. Thank you.

**Review by cah292 — APPROVED** [review]
Changes look good, unit tests pass and my concerns have been addressed. Merging.

**Pull request merged**